### ISSUE TITLE
Update "Who to follow" so that no accounts that are already followed are shown

### ISSUE THREAD

**Issue by lmurdock12** [Intro-to-SE-Spring-21/Group-8#46]
As a user if I am logged in, I do not want to see any accounts I already follow inside of the "Who to follow" block. 

Currently if I login as a given user...I can refresh the page a couple times and see the "who to follow" section is showing an account or two of people that the logged in user already follows. 

## Tasks

- [x]  #34 Should first be completed
- [x] Modify the block of code inside the views.py that randomly grabs three user accounts. so that no accounts that the logged in user already follows is added to the 'rand_three' dictionary.
  - [x] Add another condition to the while loop that checks to make sure a Follow object does not already exist where the 'user' is the currently logged in user, and the 'following' is the user that was randomly selected
- [x] Apply the changes to both the homepage and the profile page

Notes:

* Basically we need to check the 'temp' variable (which is the randomly selected user object). If the currently logged in user already follows that 'temp' user then the while loop should keep executing and selecting another 'temp' user until it finds one that the logged in user does not already follow
* https://docs.djangoproject.com/en/3.1/topics/db/queries/

## Acceptance Criteria

* For a given user, the logged in user should never see any of the accounts he already follows inside of the "Who to follow section"
* This should be implemented on the "Who to follow" section for both the homepage and the profile page.

**Comment by lmurdock12** [comment]
@kd766 @bhball22  Has this been implemented? Looks like it has when I tested on develop

**Comment by bhball22** [comment]
We are planning on still showing those that you do follow, because we have such a small list of users. So, maybe this would be a good issue for sprint 4 when have populated a larger list of users

**Comment by bhball22** [comment]
Since this has been partial completed and needs more external enhancement., I am moving this to Sprint 3
